test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -720,6 +721,28 @@ public void MAGN_7075()
             //There should be no infinite loop, otherwise, there will be an error with this test case.
         }
 
+        [Test]
+        [Category("RegressionTests")]
+        [TestModel(@".\empty.rfa")]
+        public void MAGN_7251()
```

Comment:
There are various feedback before this, is that when anyone who sees this `MAGN_7251` will have no immediate way to find out what it is about (both this test and the file names). A better way from this point onward would be to name them something like `SelectionButtonShouldBeReenabledWhenRunCompletes`. Does that make sense?